src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write your code here
+const population = document.querySelectorAll('.population');
+
+let total = 0;
+
+for (const countriePopulation of population) {
+  const stringArray = countriePopulation.innerHTML.split(',');
+  let result = '';
+
+  for (const number of stringArray) {
+    result += number;
+  }
```

Comment:
```suggestion
  const result = countriePopulation.innerHTML.split(',').join('');
```
looks like it can be simplier